src/buffer/out/AttrRow.cpp

```diff
@@ -410,7 +410,7 @@ void ATTR_ROW::ReplaceAttrs(const TextAttribute& toBeReplacedAttr, const TextAtt
             iExistingRunCoverage += pExistingRunPos->GetLength();
 
             // Copy it to the new run buffer and advance both pointers.
-            *pNewRunPos++ = *pExistingRunPos++;
+            newRun.push_back(*pExistingRunPos++);

```

Comment:
Is there a meaningful difference here between `push_back` and `emplace_back` for this type?

Reply:
No, because both will invoke the copy constructor (since `*pExistingRunPos` is of type `TextAttributeRun&`).